### PR TITLE
Add introduction for earthquake catalog

### PR DESCRIPTION
**Pull request by seisman**
Address https://github.com/seismo-learn/seismology101/issues/17

### CODE REVIEW

**Review by seisman — COMMENTED** [review]
Updates

File: source/seismology/seismic-source.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-- 最近更新日期: 2021-05-31
+- 本节贡献者: {{姚家园}}（作者）、{{田冬冬}}（作者）
+- 最近更新日期: 2022-07-13
 - 预计阅读时间: 5 分钟
 
 ---
 
-任何产生能被地震仪记录到的地球机械振动的源都可以称为震源（seismic source），可能是地震、
-火山喷发、海浪、台风或飓风、人工爆炸、交通活动等。本节介绍震源的基础概念。
+
```
Comment (seisman) [inline source/seismology/seismic-source.md:9]:
```suggestion
```

File: source/seismology/seismic-source.md [hunk 2]
```diff
@@ -57,15 +57,35 @@
...
+  ISC 地震目录通常有 24 个月的延迟。
+
+USGS PDE 地震目录
+: PDE 全称是 Preliminary Determination of Epicenters，是由美国地质调查局提供的
+  地震快速定位结果。PDE 地震目录包含了 1960 年至今的地震信息，其确定地震信息是
+  全自动且近实时的。
+
+区域地震目录有很多，通常由各个国家或区域的地震相关机构维护。比如：
+
+- 中国: [中国地震台网正式目录](https://data.earthquake.cn/datashare/report.shtml?PAGEID=earthquake_zhengshi)
```
Comment (core-man) [inline source/seismology/seismic-source.md:88]:
```suggestion
- 中国: [中国台网正式地震目录](https://data.earthquake.cn/datashare/report.shtml?PAGEID=earthquake_zhengshi)
```

File: source/seismology/seismic-source.md [hunk 3]
```diff
@@ -57,15 +57,35 @@
...
+- 日本: [JMA 地震目录](https://www.data.jma.go.jp/svd/eqev/data/bulletin/hypo_e.html)
+- 美国南加州: [SCEDC 地震目录](https://scedc.caltech.edu/eq-catalogs/)
```
Comment (core-man) [inline source/seismology/seismic-source.md:90]:
```suggestion
- 美国南加州: [SCSN 地震目录](https://scedc.caltech.edu/eq-catalogs/)
```

File: source/seismology/seismic-source.md [hunk 4]
```diff
@@ -57,15 +57,35 @@
...
-全球地震目录包含了全球范围内的地震，但通常只包含比较大的地震，小地震多有遗漏；
-区域地震目录则只关注某个特定区域内的地震，一般包含更多更小的地震。
\ No newline at end of file
+根据所包含地震的区域范围，地震目录可以分成两类：全球地震目录和区域地震目录。
+顾名思义，全球地震目录包含了全球范围内的地震，但通常只包含比较大的地震，小地震多有遗漏；
+区域地震目录则只关注某个特定区域内的地震，通常包含更多的小地震。
+
+常见的全球地震目录包括：
+
+ISC 地震目录
```
Comment (core-man) [inline source/seismology/seismic-source.md:75]:
```suggestion
International Seismological Center 地震目录
```

File: source/seismology/seismic-source.md [hunk 5]
```diff
@@ -57,15 +57,35 @@
...
+: [ISC](http://www.isc.ac.uk/iscbulletin/) (International Seismological Center)
```
Comment (core-man) [inline source/seismology/seismic-source.md:76]:
```suggestion
: [ISC](http://www.isc.ac.uk/iscbulletin/)
```

File: source/seismology/seismic-source.md [hunk 6]
```diff
@@ -57,15 +57,35 @@
...
+  地震目录是全球地震的最终版本，其涵盖了 1900 年至今的地震信息。所有的地震信息
```
Comment (core-man) [inline source/seismology/seismic-source.md:77]:
```suggestion
  地震目录是全球地震的最终版本，涵盖了 1900 年至今的地震信息。所有地震信息
```

File: source/seismology/seismic-source.md [hunk 7]
```diff
@@ -57,15 +57,35 @@
...
+  通过程序自动确定，并由 ISC 的分析人员对其进行人工检查和重定位。经过人工审核的
+  ISC 地震目录通常有 24 个月的延迟。
+
+USGS PDE 地震目录
+: PDE 全称是 Preliminary Determination of Epicenters，是由美国地质调查局提供的
+  地震快速定位结果。PDE 地震目录包含了 1960 年至今的地震信息，其确定地震信息是
+  全自动且近实时的。
```
Comment (core-man) [inline source/seismology/seismic-source.md:84]: I think [ANSS Comprehensive Earthquake Catalog](https://earthquake.usgs.gov/data/comcat/) (ComCat) is now a more popular catalog than USGS PDE. ComCat is maintained by [ANSS](https://www.usgs.gov/programs/earthquake-hazards/anss-advanced-national-seismic-system), which includes a national Backbone network, the National Earthquake Information Center ([NEIC](https://www.usgs.gov/programs/earthquake-hazards/national-earthquake-information-center-neic)), the National Strong Motion Project, and 15 regional seismic networks operated by USGS and its partners. Meanwhile, USGS PDE is maintained by NEIC, and is also a contributing catalog for ComCat. See https://earthquake.usgs.gov/data/comcat/index.php#1:

File: source/seismology/seismic-source.md [hunk 8]
```diff
@@ -51,3 +51,43 @@
...
+常见的全球地震目录包括：
+
+International Seismological Center 地震目录
+: [ISC](http://www.isc.ac.uk/iscbulletin/) 地震目录是全球地震的最终版本，涵盖了
+  1900 年至今的地震信息。所有地震信息通过程序自动确定，并由 ISC 的分析人员对其
+  进行人工检查和重定位。经过人工审核的 ISC 地震目录通常有 24 个月的延迟。
+
+ANSS ComCat 地震目录
+: ANSS Comprehensive Earthquake Catalog (ComCat) 是 ANSS (Advanced National Seismic System)
+  制作的地震目录。该目录专注于为全球 5.0 级以上地震事件提供权威和近实时的信息。
```
Comment (core-man) [inline source/seismology/seismic-source.md:82]:
```suggestion
  制作的地震目录。该目录专注于提供全球 5.0 级以上地震事件的权威、近实时信息。
```

File: source/seismology/seismic-source.md [hunk 9]
```diff
@@ -51,3 +51,43 @@
...
+除此之外，有些地震目录还提供了地震的破裂时间和震源机制等其他参数。
+
+根据所包含地震的区域范围，地震目录可以分成两类：全球地震目录和区域地震目录。
+顾名思义，全球地震目录包含了全球范围内的地震，但通常只包含比较大的地震，小地震多有遗漏；
+区域地震目录则只关注某个特定区域内的地震，通常包含更多的小地震。
+
+常见的全球地震目录包括：
+
+International Seismological Center 地震目录
+: [ISC](http://www.isc.ac.uk/iscbulletin/) 地震目录是全球地震的最终版本，涵盖了
```
Comment (core-man) [inline source/seismology/seismic-source.md:76]:
```suggestion
[International Seismological Center 地震目录](http://www.isc.ac.uk/iscbulletin/)
: ISC 地震目录是全球地震的最终版本，涵盖了
```

File: source/seismology/seismic-source.md [hunk 10]
```diff
@@ -51,3 +51,43 @@
...
+  1900 年至今的地震信息。所有地震信息通过程序自动确定，并由 ISC 的分析人员对其
+  进行人工检查和重定位。经过人工审核的 ISC 地震目录通常有 24 个月的延迟。
+
+ANSS ComCat 地震目录
```
Comment (core-man) [inline source/seismology/seismic-source.md:80]: Add the link for ComCat?

Comment (seisman) [inline source/seismology/seismic-source.md:80]:
```suggestion
[ANSS ComCat 地震目录](https://earthquake.usgs.gov/data/comcat/)
```

**Review by core-man — APPROVED** [review]

**Pull request merged**